extension/manage/js/index.js

```diff
@@ -169,4 +233,32 @@ const STATS_MAP = {
         selector: ".topCratesData",
         color: CHART_COLOR,
     });
-})();
\ No newline at end of file
+}
+
+function yearList() {
+    const y = new Date().getFullYear()
+    const year = document.querySelector(".filter-list")
+    year.addEventListener('click', function (e) {
+        if (e.target.tagName === "LI") {
+            year.childNodes.forEach(i => i.classList.remove("selected"))
+            e.target.className = "selected"
+            render(e.target.innerHTML)
+        }
+    })
+
+    for (let i = y; i > y - 3; i--) {
```

Comment:
Calculate the year range from the saved data, rather than use the current year - 3